stats/opencensus/e2e_test.go

```diff
@@ -237,6 +237,23 @@ func distributionDataLatencyCount(vi *viewInformation, countWant int64, wantTags
 	return nil
 }
 
+// waitForServerCompletedRPCs waits until both Unary and Streaming metric rows
+// appear for server completed RPC's view (by checking for length of rows to be
+// 2). Returns an error if both the Unary and Streaming metric not found within
+// the passed context's timeout.
+func waitForServerCompletedRPCs(ctx context.Context, fe *fakeExporter) error {
+	for ; ctx.Err() == nil; <-time.After(time.Millisecond) {
```

Comment:
Nit: Does 1 ms seem like aggressive for the check?

Reply:
This isn't checking anything. This is a polling loop, and this millisecond receive determines how fast to go to the next iteration of the for.

Reply:
I meant, should we go for a longer polling loop?  Im okay this way

Reply:
the polling loop is determined by the context timeout of the context passed in. In this case, that timeout is 5 seconds, the default test timeout. Thus, what this represents is within the 5 second timeout, how often to poll, which I specified as a millisecond.